src/status_im/feature_flags.cljs

```diff
@@ -0,0 +1,30 @@
+(ns status-im.feature-flags
+  (:require
+   [react-native.config :as config]
+   [reagent.core :as reagent]))
+
+(defn- enabled? [v] (= "1" v))
+
+(defn check-env [k] (enabled? (config/get-config k)))
+
+(def ^:private feature-flags-config (reagent/atom
+                           {:wallet
+                            {:edit-default-keypair (check-env :ENABLE_QUO_PREVIEW)
```

Comment:
should allow for defaults to be set by an env var so devs can have a better dev experience - not 100% this is working just yet as it was inconsistent, perhaps dependent on the app being built for it to load? 
@siddarthkay do you know?